locales/uk/LC_MESSAGES/aiogram_forms.po

```diff
@@ -0,0 +1,34 @@
+# Ukrainian translations for aiogram-forms.
+# Copyright (C) 2022 Ivan Borisenko
+# This file is distributed under the same license as the aiogram-forms
+# project.
+# Ivan Borisenko <i.13g10n@icloud.com>, 2022.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: aiogram-forms 0.4.0\n"
+"Report-Msgid-Bugs-To: i.13g10n@icloud.com\n"
+"POT-Creation-Date: 2022-10-26 00:33+0300\n"
+"PO-Revision-Date: 2022-10-26 00:24+0300\n"
+"Last-Translator: Ivan Borisenko <i.13g10n@icloud.com>\n"
```

Comment:
Thanks for contribution!
Add you as a last translator and I will merge it into next release.